### ISSUE TITLE
Playlist window displays runtime of all videos as 0:00:00

### ISSUE THREAD

**Issue by digiholic** [FredTungsten/ScriptPlayer#126]
Adding videos to the playlist shows a duration of zero, even after generating all missing files.

**Comment by FredTungsten** [comment]
Unfortunately generating files with ffmpeg doesn't influence the duration display (I wanted this to work even when ffmpeg isn't installed). The durations are generated by an external library (MediaInfo) - maybe (but unlikely) it's a codec issue. Do you have any codec megapacks installed?
(e.g. any of those. https://github.com/FredTungsten/ScriptPlayer/wiki/Troubleshooting#the-video-doesnt-load--screen-stays-black)

**Comment by digiholic** [comment]
I've went ahead and downloaded a few of those. Would I need to do anything to point ScriptPlayer at them? Or would I need to use an external player instead of local in order to get the timing? Should I download a MediaInfo dll and install it in the ScriptPlayer directory?

**Comment by FredTungsten** [comment]
To be honest I never had that problem before so I'm not sure what would help. So far it worked without any issues. ScriptPlayer is using the newest available wrapper and I'm not sure if it's compatible with newer versions of the native dll. "Playlist/Recheck all entries" just clears the list and fills it with 0:00:00 again?

**Comment by digiholic** [comment]
Yeah, rechecking just keeps it at zero. I've gone ahead and uninstalled/reinstalled scriptplayer in case the MediaInfo dll was left over from an older version or something, but it has not fixed the issue either. Do I need to run it through MPC or VLC to get any of the codec packs to work for it, or is the local player sufficient?

**Comment by digiholic** [comment]
I've just pulled the most recent source and run the program from there, and rechecking all entries generates a runtime, but I'm still unable to with the installed version, although as long as I don't recheck them in the released version, the durations will remain cached. It looks like whatever was causing this issue, it'll be resolved by the time the next release rolls out, and until then I can generate the durations in JetBrains if I need to update any.

**Comment by FredTungsten** [comment]
That's odd - the release is completely up-to-date, no changes have been made (https://ci.appveyor.com/project/FredTungsten/scriptplayer)

**Comment by digiholic** [comment]
Maybe it just had to do with the libraries I had to download in order to run the project? I'm not sure how I can get it to run in the release build though...

**Comment by FredTungsten** [comment]
Does the debug build work? (https://ci.appveyor.com/project/FredTungsten/scriptplayer/build/job/pi0r2dfgf7iyreli/artifacts)

**Comment by FredTungsten** [comment]
>Do I need to run it through MPC or VLC to get any of the codec packs to work for it, or is the local player sufficient?
Local player should work just fine. The playlist is not related to any of the external players' functionality

**Comment by digiholic** [comment]
I've got the debug build and it's still not working. In order to run the source, I had to install the Accord plugin, so I wonder if that has anything to do with it? I might be missing some sort of library that the player is looking for the executable for in the standalone version. I notice that whenever I run the recheck in the standalone, it doesn't seem to take any time at all, while when I run it in the source version, it first blanks out the runtime of everything before generating. I wonder if it's even running the recheck at all?
